usaspending_api/common/helpers/fiscal_year_helpers.py

```diff
@@ -193,6 +194,38 @@ def create_full_time_periods(min_date, max_date, group, columns):
     return results
 
 
+def _clean_subaward_spending_over_time_results(subaward_results: List[dict], date_range_type: str) -> List[dict]:
+    """Clean up the spending_over_time Subaward results
+
+    Args:
+        subaward_results: Subaward spending_over_time result for a given time period
+        date_range_type: how the results are split
+            - 'fy', 'quarter', or 'month'
+
+    Returns:
+        List of cleaned Subaward results
+    """
+
+    for result in subaward_results:
+        result["time_period"]["fiscal_year"] = result["time_period"]["fy"]
```

Comment:
It could go there, but I didn't want to any endpoint-specific code to this helper function since it could be used somewhere else in the future with a different response structure.